### DISCUSSION TITLE
Add count of closed tasks/total tasks

### DISCUSSION THREAD

**Discussion by abhinavbhagwat** [community/community#15179] [Projects and Issues]
In the Project view, it will be helpful to add the count of tasks closed/total count for a group. Today only total count is shown.

<img width="860" alt="image" src="https://user-images.githubusercontent.com/11055594/163968005-a7664fbd-7449-4b31-9724-c584808d96a3.png">
 In the above screenshot will be good to show 10/100 instead of just 100 so that we know how many tasks are closed and how many are still open under this group.

**Comment by Neudrino** [comment]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050

An **answer** from Github as of 2022-06-13 can be found at: https://github.com/github-community/community/discussions/16721#discussioncomment-2939031
